src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,60 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTests = codeWriter.getFlakyTests();
+
+      if (flakyTests.size() > 0) {
+        HashMap<TypedOperation, Integer> testOccurrences = new HashMap<>();
+        HashMap<TypedOperation, Integer> flakyOccurrences = new HashMap<>();
+
+        // Tally occurrences of all methods for if-idf
+        // Each method is counted once if it appears in a sequence test.
+        tallyOperationsInSequences(testOccurrences, explorer.getRegressionSequences());
+
+        // TODO: cxing handle Error Test Sequence tallying.
+        // tallyOperationsInSequences(testOccurrences, explorer.getErrorTestSequences());
+
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+        for (String flakyTestNum : flakyTests) {
+          int testNum = Integer.parseInt(flakyTestNum.substring(4)); // length of test
+          // Tests start at 001
+          ExecutableSequence flakyTest = regressionSequences.get(testNum - 1);
```

Comment:
The following code seems duplicative with `tallyOperationsInSequences`.  Could you make a list of flaky tests, then pass them into that method to avoid code duplication?  If that is not possible or not desirable, please document why.